cmd/destroy.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/konstructio/colony/internal/constants"
+	"github.com/konstructio/colony/internal/docker"
+	"github.com/konstructio/colony/internal/logger"
+	"github.com/spf13/cobra"
+)
+
+func getDestroyCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "destroy",
+		Short: "remove colony deployment from your host",
+		Long:  `remove colony deployment from your host`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			log := logger.New(logger.Debug)
+
+			log.Info("creating docker client")
+			dockerCLI, err := docker.New(log)
+			if err != nil {
+				return fmt.Errorf("error creating docker client: %v ", err.Error())
```

Comment:
```suggestion
				return fmt.Errorf("error creating docker client: %w ", err)
```

Always wrap errors, don't convert them to string.